crypto/crypto.go

```diff
@@ -62,3 +73,46 @@ func FromECDSAPub(pub *ecdsa.PublicKey) []byte {
 func S256() elliptic.Curve {
 	return secp256k1.S256()
 }
+
+// LoadECDSAFromString create ecdsa privatekey from string
+// ecStr should start with 0x or 0X
+func LoadECDSAFromString(ecStr string) (*ecdsa.PrivateKey, error) {
+	if len(ecStr) >= 2 && ecStr[0] == '0' && (ecStr[1] == 'x' || ecStr[1] == 'X') {
```

Comment:
I think, common method already available to check 0x prefix. If not, you can do it.
Moreover, how about define a type named Hex or HexString?

Reply:
it's not elegant too